src/please.go

```diff
@@ -194,6 +195,14 @@ var opts struct {
 		Remote bool `long:"remote" description:"Send targets to be executed remotely."`
 	} `command:"run" subcommands-optional:"true" description:"Builds and runs a single target"`
 
+	Exec struct {
+		ShareNetwork bool `long:"share_network" description:"Share network namespace"`
```

Comment:
I don't have a lot of good ideas for it, but I wonder if we want this to be more flexible? For example, you could apply the same logic to user or pid namespaces.

Reply:
Hmm yeah, I think the idea was be to add `--share_mount` as well. Not sure if there's a better approach

Reply:
Does the cli tool allow some sort of `--share=mount --share=network`?